test/Fail/Issue2446.err

```diff
@@ -1,3 +1,3 @@
 Issue2446.agda:7,1-13
-Cannot create a module containing a copy of C.primTransp
+Cannot create a module containing a copy of C.primPOr
```

Comment:
This test is _not_ broken since both the new and old error messages are valid. In fact the new error message is slightly better since `primPOr` appears before `primTransp` in the code.